lib/foreman_chef/engine.rb

```diff
@@ -33,6 +33,10 @@ class Engine < ::Rails::Engine
       ActionView::Base.send :include, ChefProxyForm
     end
 
+    initializer 'foreman_chef.dynflow_world' do |app|
+       ForemanTasks.dynflow.require!
```

Comment:
@mmoll yeah, foreman-tasks is a requirement for foreman_chef now and as @stbenjam says, we'll need this soon anyway. Once I'm done with installer support in RPM based world I can try to help with Debian. Anyway, thanks @mmoll for spotting this and all your deb packaging effort.